src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -0,0 +1,102 @@
+package com.rocketden.main.service;
+
+import com.rocketden.main.dao.RoomRepository;
+import com.rocketden.main.dto.room.CreateRoomRequest;
+import com.rocketden.main.dto.room.CreateRoomResponse;
+import com.rocketden.main.dto.room.JoinRoomRequest;
+import com.rocketden.main.dto.room.JoinRoomResponse;
+import com.rocketden.main.model.Room;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class RoomServiceTests {
+
+    @Mock
+    private RoomRepository repository;
+
+    @Spy
+    @InjectMocks
+    private RoomService service;
+
+    @Before
```

Comment:
I have something called [SonarLint](https://www.sonarlint.org/) installed that is showing the following error (see image). 
<img width="1280" alt="SonarLint Issue on `@Before` Tag" src="https://user-images.githubusercontent.com/7987279/92336679-08951e00-f058-11ea-847e-26a4f62f53e5.png">
For my PR in `ProblemTests.java`, I simply put these in a constructor because I didn't want to do `@BeforeEach` since it just needed to be run once. I'm assumming that's the same with yours -- I don't have a preference for what you want to do here, but I'm interested in your thoughts.

Reply:
Thanks, fixed this (by just deleting the method since the tests passed without it). More details below.